src/clients/authorization/AuthorizationEvaluate.tsx

```diff
@@ -90,15 +110,31 @@ export const AuthorizationEvaluate = ({
     []
   );
 
-  const evaluate = (formValues: ResourceEvaluation) => {
+  console.log(form.getValues());
+  const evaluate = async () => {
+    if (!(await trigger())) {
+      return;
+    }
+    const formValues = form.getValues();
+    const keys = formValues.resources.map(({ key }) => key);
     const resEval: ResourceEvaluation = {
       roleIds: formValues.roleIds ?? [],
+      clientId: selectedClient ? selectedClient.id! : clientId,
       userId: selectedUser?.id!,
+      resources: resources.filter((resource) => keys.includes(resource.name!)),
       entitlements: false,
-      context: formValues.context,
-      resources: formValues.resources,
-      clientId: selectedClient?.id!,
+      context: {
+        attributes: Object.fromEntries(
+          formValues.context.attributes
+            .filter((item) => item.key || item.value !== "")
+            .map(Object.values)
```

Comment:
Not sure if this works correctly, if I split this code out then `atrributes` gets an `any` type. You might want to make this a bit more explicit.
```suggestion
            .map(({ key, value }) => [key, value])
```